src/simulate/utils.py

```diff
@@ -43,7 +44,7 @@ def get_random_alpha_numeric_string(length):
 
 
 def get_current_date_time_iso8601_format():
-    iso_utc_now = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f')
+    iso_utc_now = (datetime.utcnow()+timedelta(seconds=random.randint(1,9))).strftime('%Y-%m-%dT%H:%M:%S.%f')
```

Comment:
I have added a random timestamp adds a few seconds more to caliper events when sending it.